### ISSUE TITLE
Improve Time Input

### ISSUE THREAD

**Issue by marvinrabe** [nvms/vue-atlas#26]
In my opinion the time picker is not optimal yet and I dont have a solution for it yet. I find it really hard to design one properly.

But as a first step have the same masked input like on the default time input (<input type="date">). Then it would be easier for entering times with the keyboard.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date

If you are interested I can do this for you.

**Comment by nvms** [comment]
I completely agree. I am not happy with the current Timepicker but have not yet had the inspiration for a replacement. On mobile it works nicely, as it simply defaults to the native android or iOS implementation.

Something similar to the masked date input may be a nice improvement.

**Comment by rejifald** [comment]
http://v7.carbondesignsystem.com/images/time-picker-style-3.png
I suggest something like this.
Just two or more selects, that can be combined in one input.
![image](https://user-images.githubusercontent.com/42000283/52964216-76142780-33aa-11e9-9e9f-de54c57eb586.png)
It can be look like this, but instead of date and time - hours and minutes.

**Comment by rejifald** [comment]
Any updates here ?

**Comment by nvms** [comment]
no, and this is quite low priority as there is already a timepicker component in place. yes, I know, it's a terrible implementation. but you must understand that I have a full time job and do this as a side project in my free time.

I was hoping @marvinrabe would provide a solution for this, but it appears that may not be the case any longer. unless you decide to put something together, then it will be a little while before I get around to it. if you _**do**_ decide to put something together, please keep in mind that I am **very** picky and anything you submit is obviously subject to change, by me

**Comment by rejifald** [comment]
I have full-time job too. So i spend my free time for this as well.
I am asking because i want to know. I am **not** force u, or something like that.
